presto-jdbc/src/main/java/com/facebook/presto/jdbc/AbstractConnectionProperty.java

```diff
@@ -192,6 +197,28 @@ private StringMapConverter() {}
         }
     }
 
+    protected static final class CustomHeadersConverter
```

Comment:
Can we just use the `StringMapConverter` above instead?

Reply:
No StringMapConverter splits by colon and it could impact the parsing if some custom header with hostport is passed.

Reply:
How about we percent-encode the values for this field?  That way we don't need to have two different ways to pass in a map value.

Reply:
OK Tim I have updated, please let me know if it looks good